makefile

```diff
@@ -16,24 +16,28 @@ default:
 	@echo "usage: make target"
 	@echo "available targets: compile, test, clean, check, customcheck"
 
-compile: Scatt.java ScattTest.java Submission.java SubmissionTest.java
-	javac -cp .:$(JUNIT_JAR) ScattTest.java SubmissionTest.java
-	javac Scatt.java Submission.java
+compile: Scatt.java ScattTest.java Submission.java SubmissionTest.java Report.java ReportTest.java
+	javac -cp .:$(JUNIT_JAR) ScattTest.java SubmissionTest.java ReportTest.java
+	javac Scatt.java Report.java Submission.java
 
-jar: Scatt.class ScattTest.class Submission.class
-	jar -cvmf MANIFEST.MF Scatt.jar Scatt.class ScattTest.class Submission.class SubmissionTest.class
+jar: Report.class ReportTest.class
+	jar -cvmf MANIFEST.MF Scatt.jar Scatt.class ScattTest.class Submission.class SubmissionTest.class Report.jar Report.class ReportTest.class
```

Comment:
Change to "jar -cvmf MANIFEST.MF Scatt.jar Scatt.class Submission.class Report.class"